src/locales.json

```diff
@@ -357,9 +359,10 @@
       "useful": "Useful",
       "notUseful": "Not useful",
       "yourFeedback": "Your feedback...",
-      "nothingFound": "Sorry, we could not find any results.",
+      "nothingFound": "Sorry, we could not find matching results.",
       "informationNotFound": "Not found what you were looking for?",
       "wantedInformation": "What information is missing?",
+      "thanksMessage": "Thanks for your feedback. You're helping us improving our contents and the integreat app. We would be very pleased about further feedback.",
```

Comment:
Should be `You're helping us to improve...` instead of `improving`

Reply:
Thanks :+1: